File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        int result;
        double distance = Math.sqrt(xOfDart * xOfDart + yOfDart * yOfDart);
        if (distance <= 1) result = 10;
        else if (distance <= 5) result = 5;
        else if (distance <= 10) result = 1;
        else result = 0;
        return result;
    }
}

File: solutions/python/darts/1/darts.py
import math

def score(x: int, y: int) -> int:
    distance = math.sqrt(x**2 + y**2)
    if distance > 10.0: return 0
    if distance > 5.0: return 1
    if distance > 1.0: return 5
    return 10
